### ISSUE TITLE
Release 4.1.0.Beta5

### ISSUE THREAD

**Issue by benmccann** [netty/netty#3588]
Google's [grpc-java](https://github.com/grpc/grpc-java) depends on changes only available in 4.1.0.Beta5. We can't publish anything to Maven central until there's a new Netty release. I was wondering if it'd be possible to cut a 4.1.0.Beta5 soon and address any other items scheduled for 4.1.0.Beta5 in 4.1.0.Beta6 instead?

Thanks!

**Comment by nmittler** [comment]
@benmccann we're currently working on some fixes/improvements that we're finding in our benchmarking of grpc-java.  We're getting pretty close and I think this will be wrapped up soon.

@Scottmitch @normanmaurer do we have a target for when we'd like to get 4.1.0.Beta5 out the door?

**Comment by normanmaurer** [comment]
@nmittler actualy I would like to cut 4.1.0.Final very soon :)

**Comment by Scottmitch** [comment]
@nmittler - No promises but a review of https://github.com/netty/netty/pull/3579 may make the release happen sooner ;)

**Comment by nmittler** [comment]
@Scottmitch oof, sorry this fell beneath my radar ... just took a first pass!

**Comment by trustin** [comment]
I also want my DNS refactor to be in 4.1 before release candidate phase. Just need a little bit of .. time.  That should be probably all I need for 4.1. I need streaming construct for 5.0 but that's another story.

**Comment by normanmaurer** [comment]
@trustin hurry up =P

**Comment by benmccann** [comment]
https://github.com/netty/netty/pull/3579 has been merged now. Not sure if that'd unblock us from cutting an RC? Or maybe we're waiting for the DNS refactor (https://github.com/netty/netty/pull/3499) still as well?

**Comment by trustin** [comment]
I'd prefer having a couple more betas just in case we've made some mistakes in API design / package organization, etc. Will try to finish the DNS refactor soon.

**Comment by normanmaurer** [comment]
@trustin fine by me... What you think about cut another beta early next week ? I would volunteer to do the whole release dance ;)

**Comment by trustin** [comment]
@normanmaurer Sure. Fine by me!

**Comment by benmccann** [comment]
@normanmaurer another beta cut sounds great! i'd be very excited by that

**Comment by benmccann** [comment]
Any update on this? :-)

**Comment by nmittler** [comment]
+1 gRPC would like to get this out the door :)

**Comment by normanmaurer** [comment]
@benmccann @nmittler let me do it this week for sure!

**Comment by nmittler** [comment]
@normanmaurer SGTM ... thanks!

**Comment by nmittler** [comment]
@normanmaurer is this still on your radar?

**Comment by normanmaurer** [comment]
Yes :)

**Comment by benmccann** [comment]
@normanmaurer friendly weekly reminder ping :-)

**Comment by trustin** [comment]
No pressure, but I am also waiting for you as well now. Good time to
write an automatic release note generator? ;-)

**Comment by normanmaurer** [comment]
@trustin please review 
https://github.com/netty/netty/pull/3750 . After this is merged I will release. Already in process if releasing netty-tcnative-1.1.33.Fork2

**Comment by benmccann** [comment]
@normanmaurer looks like https://github.com/netty/netty/pull/3750 was merged. Wanted to check in again to see if a release might be possible now. Thanks again!

**Comment by Scottmitch** [comment]
Release has been cut.

http://netty.io/news/2015/05/07/4-1.html

**Comment by benmccann** [comment]
Thanks you guys!
